server/controllers/upload.js

```diff
@@ -0,0 +1,41 @@
+const aws = require('aws-sdk');
+const multer = require('multer');
+
+const BUCKET_NAME = 'team-gingerbread-s3bucket';
+
+aws.config.update({
+  accessKeyId: process.env.AWS_ACCESS_KEY_ID,
+  secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
+  region: 'us-east-2'
+});
+
+const s3 = new aws.S3();
+
+const uploadToS3 = (img) => {
+  const params = {
+    Bucket: BUCKET_NAME,
+    Key: img.originalname,
+    Body: img.buffer,
+  }
+  return s3.upload(params).promise();
+};
+
+exports.multerMultiUpload = multer({storage: multer.memoryStorage()}).array('images');
```

Comment:
Optional: You can also add a check to accept only image file types (jpeg/png)

Reply:
I implemented it, check out the latest commit!